src/clients/c++/library/request.h.in

```diff
@@ -780,7 +780,8 @@ class ModelControlContext {
 ///   SharedMemoryControlGrpcContext::Create(&ctx, "localhost:8000");
 ///   std::string name = "shared_memory";
 ///   std::string shm_key = "/input";
-///   ctx->RegisterSharedMemory(name, shm_key, 0, 104);
+///   int kind = 0;
```

Comment:
Perhaps turn `kind` to enum to be more descriptive.

Reply:
Done